spotify_api_call.py

```diff
@@ -0,0 +1,79 @@
+import requests
+
+
+def get_token():
+    # TODO: set client credentials as environment variables
+    client_id = '0c5ba66cb29e46e8a415a35c19033bdd'
+    client_secret = '280da5fed992434abc6598b3beadb449'
+    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+    grant_type = {'grant_type': 'client_credentials', 'client_id': client_id, 'client_secret': client_secret}
+
+    json_res = requests.post('https://accounts.spotify.com/api/token', headers=headers, data=grant_type).json()
+
+    token_type = json_res['token_type']
+    access_token = json_res['access_token']
+
+    token_str = f'{token_type} {access_token}'
+    auth = {'Authorization': token_str}
+
+    return auth
+
+
+def get_artist_id(auth):
+    endpoint = 'https://api.spotify.com/v1/search'
+    artist = input('Enter artist name: ')
```

Comment:
This will not be needed as Flask is implemented since it is part of UI. We'll need to get user input from html.